test-data/unit/check-expressions.test

```diff
@@ -1157,6 +1157,25 @@ b'%c' % (123)
 [case testUnicodeInterpolation_python2]
 u'%s' % (u'abc',)
 
+
+-- F-String
+-- --------
+
+
+[case testFStringParseOk]
+# flags: --fast-parser
+f'foobar'
+
+[case testFStringTypecheckExpression]
+# flags: --fast-parser
+# flags: --python-version 3.6
+a: str
+a = 'foo'
+b: str
+b = f'{a}bar'
```

Comment:
Yes absolutely I was in the process of adding such a test, it's working minus the variation in error output `(diff)`